CHANGELOG_iOS.md

```diff
@@ -1,5 +1,51 @@
+ - [Security] Fixed issue with notifications pop-up not being displayed when profile is downloaded as reported on HackerOne by b4dc4t. ([#43092](https://github.com/brave/brave-browser/issues/43092))
+ - [Security] Fixed address bar not displaying URL for “about:blank” pages as reported on HackerOne by b4dc4t. ([#43016](https://github.com/brave/brave-browser/issues/43016))
+ - [Security] Fixed “tel:” pop-up not being shown on the correct page as reported on HackerOne by b4dc4t. ([#42986](https://github.com/brave/brave-browser/issues/42986))
+ - Fixed crash when closing tabs in tab view while in landscape orientation. ([#42601](https://github.com/brave/brave-browser/issues/42601))
+ - Fixed issue with content filtering in local frames. ([#40649](https://github.com/brave/brave-browser/issues/40649))
+ - Fixed ads or metadata being shown in YouTube videos intermittently. ([#43454](https://github.com/brave/brave-browser/issues/43454))
+ - Fixed location not being shown in the “VPN Region Changed” card. ([#41915](https://github.com/brave/brave-browser/issues/41915))
+ - Fixed issue with “VPN Region Changed” card getting stuck on screen when quickly navigating out of the settings page. ([#41831](https://github.com/brave/brave-browser/issues/41831))
+ - Fixed the VPN advanced server selection list not being displayed after relaunching Brave. ([#40855](https://github.com/brave/brave-browser/issues/40855))
+ - Fixed losing position of video after existing Playlist. ([#42620](https://github.com/brave/brave-browser/issues/42620))
+ - Fixed not being able to add new videos in Playlist. ([#41555](https://github.com/brave/brave-browser/issues/41555))
+ - Fixed control labels being shown for the Playlist controls for iOS 16 users. ([#42509](https://github.com/brave/brave-browser/issues/42509))
+ - Fixed issue with video duration not shown correctly in the Playlist drawer view. ([#42569](https://github.com/brave/brave-browser/issues/42569))
+ - Fixed the Leo “suggest questions” button causing UI to be unresponsive when clicked. ([#43366](https://github.com/brave/brave-browser/issues/43366))
+ - Fixed failure to download items with the same filename. ([#41284](https://github.com/brave/brave-browser/issues/41284))
+ - Fixed blank screen being displayed when loading “https://dillards.com/” when shields are raised. ([#43286](https://github.com/brave/brave-browser/issues/43286))
+ - Fixed rendering websites in dark mode which set a transparent background color. ([#43214](https://github.com/brave/brave-browser/issues/43214))
+ - Fixed tabs bar becoming disappearing with 2 or more tabs. ([#42272](https://github.com/brave/brave-browser/issues/42272))
+ - Fixed the tabs bar not being scrollable for some users. ([#41562](https://github.com/brave/brave-browser/issues/41562))
+ - Fixed Readermode not working for Brave’s RSS feed intermittently. ([#43034](https://github.com/brave/brave-browser/issues/43034))
```

Comment:
Just need to re-arrange here... Think it would sound better if we use:

```
 - Fixed Readermode not working intermittently under Brave’s RSS feed. ([#43034](https://github.com/brave/brave-browser/issues/43034))
 ```